Foundation/NSSpecialValue.swift

```diff
@@ -10,7 +10,7 @@
 internal protocol NSSpecialValueCoding {
     static func objCType() -> String
     
-    init(bytes value: UnsafePointer<Void>)
+    init(bytes value: UnsafePointer<Void>?)
```

Comment:
Are there any "special values" with a zero-byte encoding? Or should this just be non-optional?